app/src/main/res/values/strings.xml

```diff
@@ -289,7 +289,7 @@
     <!-- Preference for syncing logins -->
     <string name="preferences_sync_logins">Logins</string>
     <!-- Preference for syncing tabs -->
-    <string name="preferences_sync_tabs">Tabs</string>
+    <string name="preferences_sync_tabs">Open tabs</string>
```

Comment:
Wait, this one is not OK. You're not just changing capitalization, you need a new ID for this.

Reply:
Oh right, my bad. I'll change it now.

Reply:
Done